src/Symfony/Component/Form/AbstractType.php

```diff
@@ -47,6 +48,15 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
     }
 
     /**
+     * Configure the options for this type.
+     *
+     * @param OptionsResolver $resolver The resolver for the options.
+     */
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
It is BC compatible hence the checks for configureOptions in `ResolvedFormType`. See also my comments there.